README.md

```diff
@@ -263,9 +263,14 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // The client id given to the application by OIDC provider.
                 "client_id": "irods_http_api",
 
-                // The client secret used for accessing the introspection endpoint
+                // The client secret used for accessing the introspection endpoint.
+                // Optional unless running as a protected resource.
                 "client_secret": "xxxxxxxxxxxxxxx",
 
+                // The OIDC mode the HTTP API will run as.
+                // The value has to be "client" or "protected_resource"
```

Comment:
Please update the documentation to follow the same formatting/style as `/irods_client/tls/client_server_policy`.

List the supported options and give a brief description explaining what they do and/or how they change the interface.

Reply:
Done, check descriptions to see if like.